src/firefly/java/edu/caltech/ipac/firefly/server/visualize/imagesources/IrsaMasterDataSource.java

```diff
@@ -27,18 +28,20 @@
     @Override
     public List<ImageMasterDataEntry> getImageMasterData() {
         try {
-            URL url= ImageMasterData.class.getResource("/edu/caltech/ipac/firefly/resources/irsa-image-data.json");
-            String dataStr= URLDownload.getStringFromURL(url,null);
-            JSONParser parser= new JSONParser();
-            JSONArray data= (JSONArray)parser.parse(dataStr);
+            //URL url= ImageMasterData.class.getResource("/edu/caltech/ipac/firefly/resources/irsa-image-data.json");
+            //String dataStr= URLDownload.getStringFromURL(url,null);
+           //JSONParser parser= new JSONParser();
+            //JSONArray data= (JSONArray)parser.parse(dataStr);
+            ImageSetConverter ic = new ImageSetConverter("IRSA");
```

Comment:
Please, make "IRSA" a constant and call it from the constructor.

Reply:
OK